@LinProp/LinProp.m

```diff
@@ -260,55 +260,81 @@
                 end
             end 
         end
+        function str = string(obj)
+            
+            % The plus/minus sign coded as unicode number so this
+            % source code file is not dependent on the encoding.
+            pm = sprintf(' \xB1 ');
+            
+            function varargout = dispParts(x)
+                % Using evalc(disp(x)) prints using the current format
+                % setting. We display all parts of the number as one vector
+                % so all are displayed as floats if one of them is a float.
+                x = reshape(x, [], 1); %#ok<NASGU> % Enforce a column vector so disp does not produce 'Column x' statements.
+                varargout = strsplit(strtrim(evalc('disp(x)')));
+            end
+            
+            str = cell(size(obj));
+            for ii = 1:numel(obj)
+                
+                val_real  = get_value(real(obj));
```

Comment:
There is whitespace which should be removed.

Reply:
Just line 279 or all blank lines?

Reply:
Just the space.

```
 val_real  = get_value(real(obj));

 val_real = get_value(real(obj));
```

Reply:
Oh, I complely missed that.